utils/navigator_msgs/srv/MessageDetectDeliver.srv

```diff
@@ -0,0 +1,4 @@
+string shape_color
+string shape
+---
```

Comment:
Provide comments for what are valid values for these fields. Do I put "CROSS" or "cross", "RED" or "r"?